packages/web-runtime/src/components/SidebarNav/SidebarNav.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div id="web-nav-sidebar" :class="sidebarClasses">
+    <oc-button
+      variation="inverse"
+      appearance="raw"
+      :class="toggleSidebarButtonClass"
+      class="toggle-sidebar-button oc-py-s"
+      :aria-label="$gettext('Toggle sidebar')"
+      @click="toggleSidebarButtonClick"
+    >
+      <oc-icon size="large" :name="toggleSidebarButtonIcon" />
+    </oc-button>
+    <nav
+      class="oc-sidebar-nav oc-my-l oc-px-xs"
+      :aria-label="$gettext('Sidebar top navigation menu')"
+    >
+      <oc-list>
+        <sidebar-nav-item
+          v-for="(link, index) in navItems"
+          :id="`nav-item-${index}`"
```

Comment:
please do not use the index as id-counter here, if a new item gets added id's will change

Reply:
Removed